addons/crm_iap_enrich/data/ir_cron.xml

```diff
@@ -6,7 +6,7 @@
         <field name="user_id" ref="base.user_root"/>
         <field name="state">code</field>
         <field name="code">model._iap_enrich_leads_cron()</field>
-        <field name="interval_number">1</field>
+        <field name="interval_number">24</field>
```

Comment:
We should probably update the default value of `enrich_hours_delay` to 24 hours as well.
https://github.com/odoo/odoo/blob/master/addons/crm_iap_enrich/models/crm_lead.py#L30